src/locale/i18n.ts

```diff
@@ -224,6 +225,25 @@ export async function dynamicActivate(locale: AppLanguage) {
   }
 }
 
+export function useSystemLanguage() {
+  const {appLanguage} = useLanguagePrefs()
+  useEffect(() => {
+    async function checkSystemLanguage() {
+      try {
+        const getSystemLanguage = await getLanguageAsync()
+        const systemLanguage = getSystemLanguage.replace('_', '-')
```

Comment:
I don't want to make too many changes, but the getLanguageAsync function returns a format like "zh_CN" on Android, which doesn't match the "zh-CN" format used in others code. So, I added this conversion, but it doesn't seem to work.